openstudiocore/ruby/CMakeLists.txt

```diff
@@ -19,7 +19,13 @@ add_library( init_openstudio STATIC
 )
 
 if(MSVC)
+  # TODO: JM 2019-01-04: @macumber it seems like all three targets use the same COMPILE_FLAGS,
```

Comment:
@macumber thoughts?

Reply:
So I've tried the other day  to modify CMAKE_CXX_FLAGS inside a subdirectory, then in the root CMakeList print the CMAKE_CXX_FLAGS jsut after the add_subdirectory, and it was like it was unchanged.

Reply:
I'm not crazy about changing it just because I don't know all the side effects, I'd rather set a variable and use that three times